src/ml_board/backend/websocket_api/websocket_server.py

```diff
@@ -96,30 +96,30 @@ def on_mlgym_event(data):
             if data["event_type"] in set(["experiment_status", "job_status", "experiment_config", "evaluation_result"]):
                 print("mlgym_event: " + str(data))
                 if grid_search_id not in self._room_id_to_event_storage:
-                    self._room_id_to_event_storage[grid_search_id] = EventStorageFactory.get_disc_event_storage(parent_dir=self._top_level_logging_path,
-                                                                                                                event_storage_id=grid_search_id)
+                    self._room_id_to_event_storage[grid_search_id] = EventStorageFactory.get_disc_event_storage(
+                        parent_dir=self._top_level_logging_path, event_storage_id=grid_search_id
+                    )
                 event_id = self._room_id_to_event_storage[grid_search_id].add_event(data)
-                emit('mlgym_event', {'event_id': event_id, 'data': data}, to=grid_search_id)
```

Comment:
I believe this line got deleted by mistake! (the deletion is causing this issue [#210](https://github.com/mlgym/mlgym/issues/210))